assets/scss/pages/post.scss

```diff
@@ -42,16 +40,37 @@
     margin: 0 auto;
     display: block;
   }
+
+  @media screen and (max-width:800px) {
+    font-weight: 400;
+    max-width: 34rem;
+  }
 }
 
-.post__content a {
-  font-weight: 400;
-  color: #0366d6;
-  text-decoration: none;
+.post__container {
+  display: flex;
+  flex-direction: row;
+  max-width: 50rem;
+  margin: 0 auto 0 auto;
 
-  &:hover {
-    text-decoration: underline;
+  @media screen and (max-width:800px) {
+    flex-direction: column;
+  }
+}
+
+.post__content {
+  max-width: 40rem;
+
+  a {
+    font-weight: 400;
+    color: #0366d6;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
   }
+
 }
```

Comment:
What's the reason for these styles?

Reply:
I think these are existing styles just being moved down because of the addition of `max-width: 40rem`
![image](https://user-images.githubusercontent.com/7024160/85618673-b452e280-b693-11ea-9869-b0ad5e974395.png)

But why do we need the `max-width` here?